### ISSUE TITLE
Vorta doesn't terminate in foreground mode

### ISSUE THREAD

**Issue by varac** [borgbase/vorta#204]
**Describe the bug**
Vorta doesn't terminate in foreground mode.
When I start it with `vorta --foreground`, and close the window, it will still run and don't termiate.
I need to kill the process to terminate it properly.

**To Reproduce**

1. Start a terminal
2. Run `vorta --foreground`
3. Close vorta 
4. The shell command doesn't terminate, even `Ctrl+c` doesn't end the process.
5. `pkill vorta` terminate it

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04 (I  confirmend this on Ubuntu 18.10)
- Vorta 0.6.7, python 3.6.7

**Additional context**
If appropriate include logs from

- Linux: `$HOME/.cache/Vorta/log`

```
2019-02-22 13:54:00,786 - vorta.i18n - DEBUG - Loading translation failed for ['en', 'en-US', 'en-Latn-US'].
2019-02-22 13:54:00,808 - apscheduler.scheduler - INFO - Scheduler started

```

**Comment by m3nu** [comment]
Was already discussed [here](https://github.com/borgbase/vorta/issues/132#issuecomment-459965691) and should be a simple fix. Good to have a separate issue for it.

**Comment by m3nu** [comment]
Implemented this in #206. If you want you can test. Or wait for the next update.
